pandas/tests/io/test_pytables.py

```diff
@@ -716,10 +716,9 @@ def test_put_compression(self):
             pytest.raises(ValueError, store.put, 'b', df,
                           format='fixed', complib='zlib')
 
+    @td.skip_if_no('tables', min_version='2.2')
```

Comment:
actually you can remove this, we require tables >= 3.0

Reply:
Removed altogether in last push - the module-level importorskip already prevents this from being run without ``potables``